test/integration/binary/launchers.cpp

```diff
@@ -0,0 +1,84 @@
+/**
+ * Copyright Soramitsu Co., Ltd. All Rights Reserved.
+ * SPDX-License-Identifier: Apache-2.0
+ */
+
+#include "launchers.hpp"
+
+#include <gtest/gtest.h>
+#include "common/byteutils.hpp"
+
+#include "bindings/model_crypto.hpp"
+
+using namespace boost::process;
+
+namespace binary_test {
+
+  constexpr auto cTimeToKill = std::chrono::minutes(15);
```

Comment:
quite a lot

Reply:
Such a big timeout is used intentionally in order to let binary generators pass on slow CI or fail for any reason but time out.